### ISSUE TITLE
[BUG] Connection error on MariaDB 11.5.2

### ISSUE THREAD

**Issue by alejandro-du** [cweijan/vscode-mysql#1309]
- Database type and version:
MariaDB version 11.5.2

I can connect via CLI, but not from Database Client. I get the following error:

```
Connection error!(conn:21, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 5002ms
```

**Comment by cweijan** [comment]
Sorry I was busy recently and didn't reply in time. Please check if your extension is the latest version 7.6.4.

**Comment by cweijan** [comment]
This is because the previous version 7.6.4 was not released successfully, I have re-released version 7.6.4 to fix this issue.

**Comment by alejandro-du** [comment]
Thanks a lot! Yes, it works for me now.
